docs/_includes/Statements.md

```diff
@@ -119,44 +162,45 @@ CallStmt_ =
     [ Lhs { , Lhs } ":=" ] Lhs ";"
 ````
 
-The update statement server three logical purposes.
+The update statement serves several logical purposes.
 
-First, if it matches the grammar of a ``CallStmt_`` in which the right-most ``Lhs``
-references a method and arguments to pass to it, then the ``UpdateStmt`` is a
-method call or a new allocation with an initializing method.
-These are the only ways in which a method call is allowed in an update statement.
-In particular, the result of one method call is not allowed to be used
-as an argument of another method call (as if it had been an expression).
 
-The form
+1) The form
 
 ```
 Lhs {Attribute} ";"
 ```
 is assumed to be a call to a method with no out-parameters.
 
-The form
+2) The form
 
 ```
     [ Lhs { , Lhs } ":=" ] Lhs ";"
 ```
 can occur in the ``UpdateStmt`` grammar when there is a single Rhs that
-takes the special form of a ``Lhs`` that is a call. That is the only case
+takes the special form of a ``Lhs`` that is a call;
+that is, this form matches the grammar of a ``CallStmt_``, in which the ``Lhs`` after
+the `:=` references a method and the arguments to it, corresponding to a 
+method call or a new allocation with an initializing method.
+This is the only case
 where the number of left-hand sides can be different than the number of
 right-hand sides in the ``UpdateStmt``. In that case the number of
 left-hand sides must match the number of out-parameters of the
-method that is called.
-
-Second, if no call is involved, the ``UpdateStmt`` can be a parallel
-assignment of right-hand-side values to the left-hand sides. For example
-`x,y := y,x` to swap the values of `x` and `y`. If more than one
+method that is called or there must be just one ``Lhs`` to the left of
+the `:=`, which then is assigned a tuple of the out-parameters.
+Note that the result of a method call is not allowed to be used as an argument of
+another method call, as if it were an expression.
+
+3) If no call is involved, the ``UpdateStmt`` can be a parallel
```

Comment:
I used to say "parallel assignment", but I now more frequently say "simultaneous assignment", which I have found creates less confusion in some audiences.